mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -923,6 +923,7 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		ARG 1 tickDelta
 	METHOD method_46395 extinguishWithSound ()V
 	METHOD method_46396 reinitDimensions ()V
+	METHOD method_48155 canSprint ()Z
```

Comment:
Does this not collide with `method_46743 canSprint ()Z` in `ClientPlayerEntity`?

Reply:
```suggestion
```